Wordle.Api/Wordle.Api/Controllers/PlayerController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Data;
+using Wordle.Api.Dtos;
+using Wordle.Api.Services;
+using Player = Wordle.Api.Data.Player;
+
+namespace Wordle.Api.Controllers
+{
+    [Route("[controller]")]
+    [ApiController]
+    public class PlayerController : ControllerBase
+    {
+        private readonly PlayerService _playerService;
+
+        public PlayerController(PlayerService playerService)
+        {
+            _playerService = playerService;
+        }
+
+
+        [HttpGet("{id}")] // ✅
+        public async Task<ActionResult<Player>> GetPlayerById(int id)
+        {
+            var player = await _playerService.GetPlayerById(id);
+            if (player == null) return NotFound();
+            return Ok(player);
+        }
```

Comment:
how would the requester obtain the PlayerId? I don't think they would ever be exposed to that info. Perhaps use the username as the search criteria instead? Maybe this is fine, we're not doing authentication yet, so.